.circleci/deploy-to-pantheon.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
I am ambivalent about whether the multidev should be created with the PR or not. It does solve some problems. Just talked to an agency that is making the branch, using the multidevs created for them for testing / development, and then creating the PR later when it is ready for review. I suppose that since it is helpful for us to have the PR number, it isn't that big of a leap to open a pull request and, say, click on "review requested" when it is ready.

I am vaguely dissatisfied that the workflow for the different examples has diverged on this point, but I suppose that it minor enough that it can wait until later, when I have time to implement `--pr-only` and unify the other examples.